crates/astria-core/src/primitive/v1/mod.rs

```diff
@@ -845,15 +845,15 @@ mod tests {
             .prefix(ASTRIA_ADDRESS_PREFIX)
             .try_build()
             .unwrap();
-        insta::assert_json_snapshot!(&main_address.to_raw());
+        insta::assert_json_snapshot!("main_address", &main_address.to_raw());
```

Comment:
Let's never name this `main_bech32m_address`

Reply:
I'm assuming you meant to name it `main_bech32m_address`?